src/util.cc

```diff
@@ -435,33 +435,37 @@ std::wstring Utf8ToWide(const StringPiece& u8_string) {
 
   size_t size_needed = MultiByteToWideChar(
       CP_UTF8, 0, u8_string.str_, static_cast<int>(u8_string.size()), NULL, 0);
-  if (size_needed == 0)
-    Fatal("Failed to get needed size of wide string");
+  if (size_needed == 0) {
+    Win32Fatal("Utf8ToWide", "Failed to get needed size of wide string");
+  }
 
   std::wstring w_string(size_needed, 0);
   int w_size_needed = MultiByteToWideChar(CP_UTF8, 0, u8_string.str_,
                                           static_cast<int>(u8_string.size()),
                                           &w_string[0], w_string.size());
-  if (w_size_needed != size_needed)
-    Fatal("Failed conversion to wide string");
+  if (w_size_needed != size_needed) {
+    Win32Fatal("Utf8ToWide", "Failed conversion to wide string");
+  }
 
   return w_string;
 }
 
-int GetUtf8SizeNeeded(const WStringPiece& w_string) {
+int GetUtf8SizeNeeded(const WStringPiece w_string) {
   // Make sure that the string size can be casted to an int.
   assert(w_string.size() <= (std::numeric_limits<int>::max)());
```

Comment:
> (std::numeric_limits<int>::max)()

Why the parens?

If it's from Windows.h defining a macro for min / max, then insert #define NOMINMAX before including windows.h.

No need to uglify the code for Window's sake.

Reply:
This was to make windows happy, but I can change to the #define NOMINMAX

Reply:
try if it works in CMakeLists.txt / configure.py.